### ISSUE TITLE
The approximate training time for the video generation model

### ISSUE THREAD

**Issue by SecretMG** [cure-lab/MagicDrive#52]
Hello,

Thank you for your excellent work! I would like to ask about the approximate training time for the video generation model. I followed the instructions from [this link](https://github.com/cure-lab/MagicDrive/tree/video?tab=readme-ov-file#train-magicdrive-t) and used the command scripts/dist_train.sh 8 runner=8gpus_t +exp=rawbox_mv2.0t_0.4.3 for training. However, it took me over 30 hours to train for 5000 steps. I would like to know if this is normal because it was mentioned that approximately 80,000 steps are needed for training, which would take a considerable amount of time.

Thank you very much for your help!

**Comment by flymin** [comment]
On our V100 servers, it takes about 13h for 500 steps. Please consider using the map cache as described [here](https://github.com/cure-lab/MagicDrive/blob/ad7223f69b1b8f20ca86b3a705ca12cc50da87bd/scripts/prepare_dataset.sh#L13) for speedup and use as many CPU workers as you can.

**Comment by SecretMG** [comment]
So may I ask how long did the baseline model provided in the pic take to train for roughly? I also wonder is the baseline model trained with 61 frames with sweeps and generated annotations or just 16 frames?
![image](https://github.com/cure-lab/MagicDrive/assets/46748568/e0caabcd-000b-49c4-80f4-f9d5f0633392)

**Comment by flymin** [comment]
It takes about 10 days to train with 8V100. If you have more GPUs or larger GPU mem, the training should take shorter time by adjusting the batch size and learning rate.

The baseline model only train with 16-frame generation. We train another model for 61-frame generation with the new configuration.
